From 8bd5aaa7d13f0a86f64071775b39522d17825d05 Mon Sep 17 00:00:00 2001
From: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Date: Mon, 20 May 2024 14:05:00 +0200
Subject: [PATCH 0865/1695] dts: stm32: add support of m33_rproc for
 stm32mp215f-dk board

Enable m33 rproc for stm32mp215f-dk board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f167d3b39a747ae3a74ec01d6161035d82b9dd45 dts: stm32: add support of m33_rproc for stm32mp215f-dk board]
Change-Id: I8f85044db8969fc359f3e8f631e2948b113dd260
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/383220
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index 00e6cafd4..e11fb7213 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -69,6 +69,10 @@
 	status = "okay";
 };
 
+&m33_rproc {
+	status = "okay";
+};
+
 &pka {
 	status = "okay";
 };
-- 
2.47.2

